### DISCUSSION TITLE
two problems in use

### DISCUSSION THREAD

**Discussion by Cheney-77** [jertel/elastalert2#1296] [Q&A] [answered]
1.does ea2 support deploy in cluster mode?
2.can we write multi rules in one xxx_rule.yaml?

**Comment by jertel** [comment, marked as answer]
1. No, you can search these forums for "HA" or "high availability" as there are several questions and answers on this topic.
2. No, it's one rule per file.

**Reply by Cheney-77** [reply]
ok,thank you so much

**Reply by jertel** [reply]
You're welcome. Also, if you are using Kubernetes there are some additional options for packaging all rules into a single configmap file.
